cmds/helminstaller/README.md

```diff
@@ -0,0 +1,52 @@
+# Helm Installer for the TOI Framework
+
+The helm installer provides an executor plugin for the [TOI framework](../../docs/reference/ocm_toi-bootstrapping.md).
+It can be used to install a top level helm chart, which might optionally
+by composed of a set of sub charts.
+
+## Component
+
+The helm installer plugin for the TOI framework if provided
+by component `ocm.software/toi/installers/helminstaller`.
+
+It provides a single `toiExecutor` resource with the
+name `toiexecutor`.
+
+## Configuration
+
+The executor configuration supports the following field:
+
+- `chart` *[ResourceReference](../../docs/reference/ocm_toi-bootstrapping.md#resourcereference)* (**required**) The resource containing the top level hemlm chart to work on.
+
+- `subCharts` *map[string]ResourceReference* (**optional**) A set of resources describing charts used as sub charts for the top level chart.
```

Comment:
Documentation is insufficient here:

- What is the meaning of the key in this map or why is it not an array of ResourceReferences?
Example
```
        foo:
          resource:
            name: nginx-controller-chart
          referencePath:
          - name: nginx-controller
        bar:
          resource:
            name: echo-chart
          referencePath:
          - name: echoserver
```
What is the meaning of foo/bar?

What happens with imageMapping for subcharts?

Example would be helpful.

Reply:
It uses the value propagation mechanism of the helm subchart mechanism. See https://helm.sh/docs/chart_template_guide/subcharts_and_globals/. So, there is nothing special.
I've added a short description to the README.

The key of the subchart map is the name of the subchart folder below the parent chart's `charts` folder.
I also added this to the description.